index.php

```diff
@@ -52,10 +52,14 @@ function adminer_object()
 
 		//new AdminerISPConfig('admin', 'admin', 'https://server.domain.tld', true),
 
-		new AdminerBootstrapLike(false),
 	];
 
-	return new AdminerPlugin($plugins);
+	$adminer = new AdminerPlugin($plugins);
```

Comment:
Please, make the AdminerBootstrapLike be the main initializer.

Extends the AdminerPlugin and do the propagation checks in the AdminerBootstrapLike.